# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/hannes/checkpassword/print-cc-head.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./print-cc.sh.orig	2004-09-12 02:13:40.931293208 +0200
+++ ./print-cc.sh	2004-09-12 02:14:12.149547312 +0200
@@ -1,4 +1,4 @@
-cc="`head -1 conf-cc`"
+cc="`head -n 1 conf-cc`"
 systype="`cat systype`"
 
 cat warn-auto.sh
